### ISSUE TITLE
Cannot read property 'getConnectivityState'

### ISSUE THREAD

**Issue by marckaraujo** [hyperledger/composer#3948]
## Context
When I run: `composer runtime install -c adminCard -n proak-hyperledger-network`

I get this error:
```
/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/fabric-client/lib/EventHub.js:308
			if(self._stream) state = self._stream.call.channel_.getConnectivityState();
			                                                    ^

TypeError: Cannot read property 'getConnectivityState' of undefined
    at ClientDuplexStream.<anonymous> (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/fabric-client/lib/EventHub.js:308:56)
    at emitOne (events.js:116:13)
    at ClientDuplexStream.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at ClientDuplexStream.Readable.push (_stream_readable.js:208:10)
    at Object.onReceiveMessage (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/grpc/src/client_interceptors.js:1302:19)
    at InterceptingListener.recvMessageWithContext (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/grpc/src/client_interceptors.js:629:19)
    at /home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/grpc/src/client_interceptors.js:728:14
```
## Expected Behavior
It should install the network

## Steps to Reproduce
`composer runtime install -c adminCard -n proak-hyperledger-network`

## Your Environment
Node.js version: v8.9.1
(http_parser: 2.7.0, v8: 6.1.534.47, uv: 1.15.0, zlib: 1.2.11, ares: 1.10.1-DEV,
 modules: 57, nghttp2: 1.25.0, openssl: 1.0.2m, icu: 59.1, unicode: 9.0,
 cldr: 31.0.1, tz: 2017b)

node-report version: 2.2.1 (built against Node.js v8.9.1, glibc 2.23, 64 bit)

OS version: Linux 4.4.0-1052-aws #61-Ubuntu SMP Mon Feb 12 23:05:58 UTC 2018
(glibc: 2.23)

composer-cli@0.18.1

**Comment by marckaraujo** [comment]
This error is due to #3944. 
But I can't upgrade to branch master since Hyperledger Fabric from IBM only accepts 0.18.1.
I have manually lock the grpc version on the modules, but seems that need to do it in more than one place.

Could you help with it?
It now is showing for me:

```
Error: Failed to load connector module "composer-connector-hlfv1" for connection type "hlfv1". Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.-Must create ConnectionManager with a ConnectionProfileManager implementation.
    at Promise.resolve.then (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-rest-server/node_modules/composer-common/lib/connectionprofilemanager.js:144:42)
```

**Comment by vmorris** [comment]
I do not believe this is a composer issue. I am duplicating the same error in PTE..
```
info: [PTE 0 main]: [channelAddEvent] requests: grpcs://n9208bb8db3e946cca0d452fe6cddf92e-org1-peer1.stage.blockchain.ibm.com:31002, events: grpcs://n9208bb8db3e946cca0d452fe6cddf92e-org1-peer1.stage.blockchain.ibm.com:31003
/home/testuser/fabric-sdk-node/fabric-client/lib/EventHub.js:308
                        if(self._stream) state = self._stream.call.channel_.getConnectivityState();
                                                                            ^

TypeError: Cannot read property 'getConnectivityState' of undefined
    at ClientDuplexStream.<anonymous> (/home/testuser/fabric-sdk-node/fabric-client/lib/EventHub.js:308:56)
    at emitOne (events.js:116:13)
    at ClientDuplexStream.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at ClientDuplexStream.Readable.push (_stream_readable.js:208:10)
    at Object.onReceiveMessage (/home/testuser/fabric-sdk-node/node_modules/grpc/src/client_interceptors.js:1302:19)
    at InterceptingListener.recvMessageWithContext (/home/testuser/fabric-sdk-node/node_modules/grpc/src/client_interceptors.js:629:19)
    at /home/testuser/fabric-sdk-node/node_modules/grpc/src/client_interceptors.js:728:14
```

**Comment by marckaraujo** [comment]
@vmorris, Yes, This error is due to #3944.

I just added the grpc 1.10.1 to the package.json and now it is working.

**Comment by vmorris** [comment]
Yep, confirmed that as well -- thanks.

**Comment by sstone1** [comment]
Duplicate of #3944.

**Comment by sridharraoy** [comment]
Team, I got the same problem with grpc 1.11.3. But when i rolled back grpc to 1.10.1 it started working.

**Comment by ankitdeo** [comment]
Yes, got the same issue resolved when I upgraded grpc 1.6.0 to 1.10.0 . These kind of issues eat a lot of time :)

**Comment by nyetwurk** [comment]
Why is this issue closed?
